tests/UKHO.S100PermitService.API.FunctionalTests/Helpers/TestBase.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using NUnit.Framework;
+using UKHO.S100PermitService.API.FunctionalTests.Configuration;
+
+namespace UKHO.S100PermitService.API.FunctionalTests.Helpers
```

Comment:
Move to UKHO.S100PermitService.API.FunctionalTests namespace, and remove 'Helpers' namespace?

Reply:
Resolved. Thanks